.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Pull Request Template
+about: Template for submitting a Pull Request
+---
+
+<!--
+  To make it easier for us to help you, please include as much useful information as possible.
+
+  Useful Links:
+  - How to Contribute: hhttps://github.com/Pand-Aid/pandaid-api/docs/development/contributing.md
+  - How to File an Issue: https://github.com/Pand-Aid/pandaid-api/docs/development/how-to-file-issue.md/
+
+  Before opening a new issue, please search existing issues:  https://github.com/Pand-Aid/pandaid-api/issues
+-->
+
+**IMPORTANT: Please do not create a Pull Request without creating an issue first.**
+
+## What does this do and why?
+
+Explanation of what the ticket is accomplishing. E.g. What endpoints are you adding, why does this change need to be made, why this refactor needed, etc.
+
+Please link to Github issue
+
+## Additional notes for reviewers
+
+Place for gotchas, expectations for how code changes should operate, or anything else that you would like your reviewer to know that was not covered in the “Why” section.
+
+## Pre-Review Tasks
+
+Place an ‘x’ between the brackets.
```

Comment:
Is there an expectation that all brackets need checked? I think this implies that they need to be and so if we're not too stringent on that we should be more explicit about it. Or we do want to ensure that all are included?